src/tellor_disputables/cli.py

```diff
@@ -29,15 +30,22 @@ def print_title_info() -> None:
 
 
 @click.command()
-@click.option("-a", "--all-values", is_flag=True, show_default=True)
+@click.option("-a", "--all-values", is_flag=True, show_default=True, help="if set, get alerts for all values")
 @click.option("-w", "--wait", help="how long to wait between checks", type=int)
+@click.option("-f", "--filter", help="build a feed and get alerts for that feed only", is_flag=True)
```

Comment:
confusing use of "feed" when you're  filtering based on query id

Reply:
good catch